src/main/java/com/sprint/mission/discodeit/service/jcf/JCFChannelRepository.java

```diff
@@ -0,0 +1,40 @@
+package com.sprint.mission.discodeit.service.jcf;
+
+import com.sprint.mission.discodeit.entity.Channel;
+import com.sprint.mission.discodeit.repository.ChannelRepository;
+
+import java.util.*;
+
+public class JCFChannelRepository implements ChannelRepository {
+    private final Map<UUID, Channel> data = new HashMap<>();
+
+    @Override
+    public Channel create(Channel channel) {
+        data.put(channel.getChannelId(), channel);
+        return channel;
+    }
+
+    @Override
+    public Channel read(UUID id) {
+        return data.get(id);
+    }
+
+    @Override
+    public List<Channel> readAll() {
+        return new ArrayList<>(data.values());
```

Comment:
방어적 복사로 반환값 래핑 잘 해주셨네요 💯   
new ArrayList<>로 반환하면 호출처에서 수정할 수 있으니 불변으로 바꿔볼까요?  
```suggestion
        return List.of(data.values());
```